indico/htdocs/sass/modules/_users.scss

```diff
@@ -78,10 +78,14 @@
     display: inline-block;
     font-size: 0.95em;
     padding: 0.2em 0.5em;
-    margin-bottom: 0.1em;
+    margin-bottom: 0.5em;
     color: $black;
     background: $pastel-gray;
 
+    &.one-line {
```

Comment:
I don't think this belongs in here. If you want to have one per line simply target `.author-list li` within the specific box in `_abstract_page.scss`.